lib/server.js

```diff
@@ -253,7 +253,20 @@ function Server(opts, callback) {
   that.on("subscribed", function(topic, client) {
     that.publish({
       topic: "$SYS/" + that.id + "/new/subscribes",
-      payload: client.id
+      payload: JSON.stringify({
```

Comment:
these are not in master! seems they are blown. This makes https://github.com/mcollina/mosca/pull/315 not to pass